fgd/point/linked_portal_door.fgd

```diff
@@ -8,5 +8,6 @@
 	width(integer) : "Half-Width (G)" : 64 : "Half the width of the portal, on the Green axis."
 	height(integer) : "Half-Height (B)" : 64 : "Half the height of the portal, on the Blue axis."
 	isstatic(boolean) : "Static Portal" : 0 : "If set to true, this portal does not ever move or toggle, and allows VRAD to cast light through it."
-	startactive(boolean) : "Start Active" : 0 : "Whether to start the linkage as active from the start."
+	startactive[+P2CE](boolean) : "Start Active" : 0 : "Whether to start the linkage as active from the start."
+	startactive[+MOMENTUM](boolean) : "Start Active" : 1 : "Whether to start the linkage as active from the start."
```

Comment:
Good to think about other games when doing this - this is a momentum specific thing so should only be true for momentum and false for every other game. [func_button](https://github.com/ChaosInitiative/Chaos-FGD/blob/master/fgd/brush/func/func_button.fgd#L21-L22) has a good example of this

Reply:
Alternatively just do an untagged one, and one with `[MOMENTUM]` - untagged ones are overridden by tagged ones.

Reply:
! Good to know! Thanks spen :D